acados/ocp_nlp/ocp_nlp_sqp_rti.h

```diff
@@ -57,36 +57,41 @@ extern "C" {
  * options
  ************************************************/
 
+
 typedef struct
 {
 	ocp_nlp_opts *nlp_opts;
     int compute_dual_sol;
-    int ext_qp_res;      // compute external QP residuals (i.e. at SQP level) at each SQP iteration (for debugging)
-    int qp_warm_start;   // NOTE: this is not actually setting the warm_start! Just for compatibility with sqp.
+    int ext_qp_res;           // compute external QP residuals (i.e. at SQP level) at each SQP iteration (for debugging)
+    int qp_warm_start;        // NOTE: this is not actually setting the warm_start! Just for compatibility with sqp.
     bool warm_start_first_qp; // to set qp_warm_start in first iteration
-    int print_level;     // possible values 0, 1 
+    int phase;                // phase of RTI. Possible values 0, 1 and 2.
```

Comment:
I would specify a bit better in the comment what phase of RTI is (i.e. feedback/preparation).

In general the name `phase` is confusing as it may refer to multi-phase, which is something we want to implement anyway at some point.
What about renaming `phase` into `rti_phase`?

Reply:
I agree (addressed 064a5ff)